services/core-web/src/styles/components/MajorMineApplication.scss

```diff
@@ -0,0 +1,5 @@
+.decision-package-uploads {
```

Comment:
This form item label is misaligned because it's using the old antd Form.Item from "@ant-design/compatible". Now, I'm not _sure_ because I can't see the DOM structure right now, but there is a very good chance that you can straight up _remove_ that Form.Item wrapper because it'll have another Form.Item nested within it from the RenderFileUpload component. I would try that first, and if it doesn't work, just update the import to be from "antd" instead.

Reply:
Thanks I will try this out!